docs/included.md

```diff
@@ -116,7 +116,7 @@
 * - Quantile Interval Score
   - [API](api.md#scores.continuous.quantile_interval_score)
   - [Tutorial](project:./tutorials/Quantile_Interval_And_Interval_Scores.md)
-  - [Gneiting and Raftery (2007) - Corollary 5.2](https://doi.org/10.1198/016214506000001437)
+  - [Gneiting and Raftery (2007) - Section 6.2](https://doi.org/10.1198/016214506000001437)
```

Comment:
For this, Winkler 1972 is a better reference. I think the Gneiting and Raftery paper focuses on the interval score only.
https://doi.org/10.2307/2284720

Reply:
done